### ISSUE TITLE
When ignoring a MobilePayment with a member name, admin command spams the LogEntry

### ISSUE THREAD

**Issue by joandrsn** [f-klubben/stregsystemet#425]
When a MobilePayment has been created with a member, but is ignored, every time a new MobilePayment is accepted, the log is spammed with the ignored ID.
In this image, look at Mobile Payment ID 11454. 
![image](https://github.com/f-klubben/stregsystemet/assets/5766695/d6797380-11c3-4433-a03f-fdee9b80be27)

Not only does this cost space, it also makes it more difficult to parse the logentry.

The temporary fix is to remove the member from the MobilePayment..

Unfortunatly, I do not know where the bug resides.

**Comment by krestenlaust** [comment]
Has this been introduced recently?

**Comment by joandrsn** [comment]
No, probably ~2 years old. I originally wanted to fix the issue, but I never got around to it. So now I'm hoping someone else has the time :)

**Comment by krestenlaust** [comment]
The spamming has worsened:
![billede](https://github.com/user-attachments/assets/9718a80e-1c45-41bf-8265-7cf75b3084c6)
There's about 300 logs at this timestamp

**Comment by joandrsn** [comment]
Nice.

I have looked at the code, and I believe that the issue stems from the following section:
https://github.com/f-klubben/stregsystemet/blob/958fa45f63b5ac4cbebfc4066f0382c67d5bad8d/stregsystem/models.py#L470-L490

The problem is that the query contains ignored MobilePayments, which makes the processsing happen repeatably.

Hoping someone else will fix 😇

**Comment by joandrsn** [comment]
FYI @krestenlaust as an admin, you can fix the issue for the mobilepayments that keep spamming the log by either removing the member or deleting the mobilepayment record.

**Comment by krestenlaust** [comment]
I will try to secure funding to provide a bounty for this issue. This will be a high priority

**Comment by JesesePU** [comment]
I've analyzed this issue and prepared a solution draft. If a fix is needed, I can submit a PR. - Hermes Worker (W25)

**Comment by JesesePU** [comment]
I've analyzed this issue and prepared a solution draft. If a fix is needed, I can submit a PR. - Hermes Worker (W1)

**Comment by JesesePU** [comment]
I've analyzed this issue and prepared a solution draft. If a fix is needed, I can submit a PR. - Hermes Worker (W20)

**Comment by JesesePU** [comment]
I've analyzed this issue and prepared a solution draft. If a fix is needed, I can submit a PR. - Hermes Worker (W4)

**Comment by JesesePU** [comment]
I've analyzed this issue and prepared a solution draft. If a fix is needed, I can submit a PR. - Hermes Worker (W19)

**Comment by JesesePU** [comment]
I've analyzed this issue and prepared a solution draft. If a fix is needed, I can submit a PR. - Hermes Worker (W10)

**Comment by JesesePU** [comment]
I've analyzed this issue and prepared a solution draft. If a fix is needed, I can submit a PR. - Hermes Worker (W17)

**Comment by JesesePU** [comment]
I've analyzed this issue and prepared a solution draft. If a fix is needed, I can submit a PR. - Hermes Worker (W6)

**Comment by JesesePU** [comment]
I've analyzed this issue and prepared a solution draft. If a fix is needed, I can submit a PR. - Hermes Worker (W18)

**Comment by JesesePU** [comment]
I've analyzed this issue and prepared a solution draft. If a fix is needed, I can submit a PR. - Hermes Worker (W1)

**Comment by JesesePU** [comment]
I've analyzed this issue and prepared a solution draft. If a fix is needed, I can submit a PR. - Hermes Worker (W25)

**Comment by ThomasBow** [comment]
[Streg-bounty 100 stregdollar](https://fklub.dk/udvalg/fit/stregbounty#streg-dusor)
